Dockerfile

```diff
@@ -82,7 +81,7 @@ COPY src/cycles_ledger/client/Cargo.toml src/cycles_ledger/client/Cargo.toml
 COPY src/cycles_ledger/pic/Cargo.toml src/cycles_ledger/pic/Cargo.toml
 COPY src/cycles_ledger/types/Cargo.toml src/cycles_ledger/types/Cargo.toml
 COPY src/shared/Cargo.toml src/shared/Cargo.toml
```

Comment:
We use the same target directory as for local builds, so that we can have the same code paths and eliminate branches.